src/runtime/schema/settings.ts

```diff
@@ -57,6 +57,10 @@ export type SettingsInput = {
     // being tracked here: https://github.com/microsoft/TypeScript/issues/13195
     [connectionTypeName: string]: ConnectionSettings | undefined | false
   }
+  /**
+   * Disable or configure the authorization plugin
```

Comment:
Wondering how to balance the difference between "go to this other place to read about a plugin" vs inline docs that may not even mention the plugin.

Reply:
Ideally we should have both. A shorten link that points to the full docs + an inline description that helps people in the 80% cases

Reply:
Maybe you see this differently than me but my view is this is 1) super needed 2) experimental stage (product wise). So, my feeling is we can keep things pretty raw for now, including docs.

Reply:
I have the same feeling. It's something along the lines of "I want it to be there but don't want to make too much noise about it yet".